basiclti/basiclti-portlet/src/webapp/WEB-INF/sakai/IMSBLTIPortlet.xml

```diff
@@ -1457,8 +1457,8 @@
           <configuration name="final.releaseemail" value="true"/>
           <configuration name="final.custom" value="true"/>
           <configuration name="final.allowsettings" value="true"/>
-          <configuration name="final.allowroster" value="true"/>
-          <configuration name="final.allowoutcomes" value="true"/>
+          <configuration name="final.allowroster" />
+          <configuration name="final.allowoutcomes" value="false"/>
           <configuration name="final.allowlori" value="true"/>
```

Comment:
`final` configurations control what fields are available by the instructor/admin at launch time.  I believe want to turn off the `imsti` settings.